ec3730/Controllers/PingViewController.swift

```diff
@@ -315,6 +315,9 @@ struct PingSwiftUIViewController: View {
     @State var showAlert: Bool = false
     @State var alertMessage: String?
 
+    @State var isActive = false
+    @FetchRequest(fetchRequest: PingSet.fetchAllRequest()) var pings: FetchedResults<PingSet>
```

Comment:
Does this slow down page loading at all?

Reply:
Now I implemented limit, number of requests are fetching according to the limit. If limit is not set, then all requests will be fetched.

Reply:
Excellent!